app/components/App.js

```diff
@@ -3,6 +3,95 @@ import Navigation from './Navigation';
 import CategoryPage from './Find/FindPage';
 import ResourcesTable from './Search/ResourcesTable';
 
+/**
+ * Get user location.
+ *
+ * Makes use of both the HTML5 Geolocation API and the Google Maps Geolocation
+ * API. Currently restricts the location to within San Francisco to avoid
+ * inaccurate geolocation results, but this should be removed if more locations
+ * are added.
+ *
+ * @returns A Promise of a location, which is either an object with `lat` and
+ * `lng` properties or an error if location is unavaible or out of bounds.
+ */
+function getLocation() {
```

Comment:
Typically in React, functions are placed within the component. We don't follow this practice throughout the app but this is a small change that could be made.